bin/diagnostic.js

```diff
@@ -1 +1,23 @@
 'use strict';
+
+```js
+//We'll learn about require later in the course
+const ask = require('../lib/ask.js');
+
+let answer = 'not empty';
+
+while (answer !== '7') {
+  answer = ask("Guess my secret? ");
+  if (answer = [i];
```

Comment:
Where is [i] defined? You were close but you need to reference an answer value and validate off of that.